opencog/nlp/aiml/import/aiml2psi.pl

```diff
@@ -545,17 +549,24 @@ sub process_tag
 
 	$text =~ /(.*?)<$tag>(.*?)<\/$tag>(.*)/;
 
-	$tout .= &process_aiml_tags($indent, $1);
+	push(@pt1, $1);
+	push(@pt2, $2);
+	push(@pt3, $3);
+
```

Comment:
Oh right that would be better, I'll update it and hunt down the rest in the code before the demo